dictionaries/python/src/data_science.txt

```diff
@@ -94,3 +94,27 @@ upsample
 parallelizable
 ONNX
 enwik
+Hadamard
+Schur product
+rsqrt
+bucketizes
+tera
+strided
+invvar
+fastai
+behaviour
+nesterov
+sutskever
+perceptron
+SATA
+randint
+elif
+linspace
+constexpr
+depanding
+amsgrad
+NVLAMB
+volumn
```

Comment:
This is a misspelling of volume. I would rather not include `volumn` if it can be avoided.

Reply:
you're right , revert done